src/Arcus.Observability.Telemetry.Core/Logging/RequestLogEntry.cs

```diff
@@ -32,6 +33,7 @@ public class RequestLogEntry
             string method, 
             string host, 
             string uri, 
+            string operationName,
```

Comment:
Technically, this is a breaking change, as it's a `public` class with a `public` ctor. We should consider making an overload instead.

Reply:
I added an overload, see https://github.com/arcus-azure/arcus.observability/pull/229/commits/13c77ee974d22ae0d69215b57ad32405118f33e5